content/tutorials/getting-started/index.md

```diff
@@ -173,53 +179,40 @@ Gatling Enterprise requires a package built from a Gatling project in order to r
 To create a package, run the `enterprisePackage.bat` (if Windows) or `enterprisePackage.sh` (if Linux/MacOS) file, Gatling will start creating your first package:
 
 ```
-$ ./bin/enterprisePackage.sh
-GATLING_HOME is set to .../gatling-charts-highcharts-bundle-{{< var gatlingVersion >}}
-GATLING_VERSION is set to '{{< var gatlingVersion >}}'
-Creating jar...done
+$ ./bin/gatling.sh
 ```
 
-This script can also be clicked or double clicked from any file explorer. You'll see the result in the folder `target`, which should now have a file named `package.jar`.
-
-{{< youtube _fq3giAIibw >}}
+And now type `3` to package your classes. You'll see the result in the folder `target`, which should now have a file named `package.jar`.
 
 #### Step 3.2 - Upload the Package {#upload-package}
 
 On Gatling Enterprise, click on the [Packages section]({{< ref "../../reference/user/package_conf/" >}}) and on create. Give a name to your package, you can choose to make it available to all teams or to limit access to a specific one, then hit save.
 
 Next, upload the package you generated earlier by clicking on the cloud icon. The package you generated earlier is located here: `target/package.jar`
 
-Upload it to Gatling Enterprise, either by drag-and-dropping it to the modal, or by clicking on the modal to open the file manager.
+Upload it to Gatling Enterprise, by clicking on the upload button on the right side then either drag-and-drop your file or by clicking on the modal to open the file manager.
```

Comment:
I don't remember this UI...
But I think it is:

```suggestion
To upload it to Gatling Enterprise, click on the upload button on the right side that will open a modal.
Either drag-and-drop your file or open the file manager by clicking on the modal.
```

I'm bothered by the fact that the "by clicking on the modal" is not in the same tone as "drag-and-drop your file".

WDYT?

Reply:
Yep was weird